curriculum/challenges/english/14-responsive-web-design-22/learn-the-css-box-model-by-building-a-rothko-painting/60a3e3396c7b40068ad6997f.md

```diff
@@ -13,10 +13,10 @@ Replace the `padding` property with `overflow` set to `hidden` - changing the ca
 
 # --hints--
 
-You should remove the `padding` property from the `.one` selector.
+You should remove the `padding` property from the `.canvas` selector.
 
 ```js
-const onePadding = new __helpers.CSSHelp(document).getStyle('.one').getPropertyValue('padding');
+const onePadding = new __helpers.CSSHelp(document).getStyle('.canvas').getPropertyValue('padding');
 assert(!onePadding);
```

Comment:
Small nitpicking, it'd be nice to have `onePadding` name changed as well for consistency.

Reply:
Done!